### ISSUE TITLE
mitochondrial DNA depletion syndrome terms missing definitions

### ISSUE THREAD

**Issue by sbello** [DiseaseOntology/HumanDiseaseOntology#692]
Many of the mitochondrial DNA depletion syndrome terms are missing definitions. Please review and add definitions.
Also reopened ticket #177 to review merging 4a with Alpers syndrome.

**Comment by lschriml** [comment]
Michael,
 (1) create a parent term: 
        mitochondrial DNA depletion syndrome
            for 
             https://www.omim.org/phenotypicSeries/PS603041

(2) for the subtypes:  (types 1-15)
       -- fill in the needed definitions and update definitions with the new parent term, where  
           needed.

        For each definition: include the two parts: 'characterized by' and 'has_material_basis_in' 

(3) Check out the Genetics Home Reference link, for each subtype
          -- identify synonyms, and include the GHR link as a xref on the definition 

      -- add the Orphanet ID

(4) In the same branch: 
   add a new parent term for : 
      multiple mitochondrial dysfunctions syndrome
          for https://www.omim.org/phenotypicSeries/PS605711
     Update the DO terms: -- create definitions, as above
       multiple mitochondrial dysfunctions syndromes (types 1-6)

Cheers,
Lynn

**Comment by msinclair2** [comment]
Lynn and Sue,

I've made Lynn's requested additions above.  Mitochondrial DNA depletion syndrome and multiple mitochondrial dysfunctions syndrome are now parent terms under which all the individual syndromes are placed.  All syndrome terms have definitions, synonyms, cross-references, including GHR and Orphanet where available.

Thank you Sue for bringing these to our attention!

I did not merge Alpers syndrome with mitochondrial DNA depletion syndrome 4a in this update, but according to my research on this term, it really seems to me that Alpers syndrome and mitochondrial DNA depletion syndrome 4a really are the same disease and the terms should be merged, with the Alpers syndrome term being deprecated (#177).
